### PR TITLE
add proposed Math extensions

### PR DESCRIPTION
**Pull request by dnalborczyk**
stage 1 proposal for Math extensions.

tests are based on ES6 "Number properties" and "Math methods", significance: small.

https://github.com/tc39/proposals
https://github.com/rwaldron/proposal-math-extensions

### CODE REVIEW

File: data-esnext.js [hunk 1]
```diff
@@ -1686,6 +1686,56 @@ exports.tests = [
...
+      name: 'Math.degrees',
+      exec: function() {/*
+        return typeof Math.degrees === "function";
+      */},
+      res: {},
+    },
+    {
+      name: 'Math.RAD_PER_DEG',
+      exec: function() {/*
+        return typeof Math.RAD_PER_DEG === "number";
```
Comment (ljharb) [inline data-esnext.js:1726]: these should check the values as well, not just the types.

File: data-esnext.js [hunk 2]
```diff
@@ -1686,6 +1686,56 @@ exports.tests = [
...
+      name: 'Math.radians',
+      exec: function() {/*
+        return typeof Math.radians === "function";
+      */},
+      res: {},
+    },
+    {
+      name: 'Math.degrees',
+      exec: function() {/*
+        return typeof Math.degrees === "function";
```
Comment (ljharb) [inline data-esnext.js:1719]: we should also invoke each of these for some edge cases and example values, and not just test presence.

**Comment by zloirock** [comment]
https://github.com/kangax/compat-table/commit/e87b4db995ca3e4ed134671908e32c36799f3f68
